docs/docker.md

```diff
@@ -74,41 +86,41 @@ Data used for the image is in hieradata/role/$puppetrole.yaml
 
 ### General maintenance
 
-A few other commands are available for general Docker maintenance.
+A few other commands are available for general ```Docker``` maintenance.
 
 
 #### Docker status
 
-To show general Docker information (version, containers and images):
+To show general ```Docker``` information (version, containers and images):
 
     fab docker.status
     bin/docker_status.sh
 
 #### Docker files cleanup
 
-To remove all local images and containers (WARNING: have no important data there).
+To remove all local images and containers (**WARNING:** have no important data there).
 
 By default a confirmation prompt appears:
 
     fab docker.purge
     bin/docker_purge.sh
 
-To run in unattended mode (useful for cleanups in CI pipelines):
+To run in unattended mode (useful for cleanups in ```CI pipelines```):
 
     fab docker.purge:auto
     bin/docker_purge.sh auto
 
 
 ### Docker prerequisites
 
-Docker operations via Fabric or using the command line require Docker to be locally installed.
+```Docker``` operations via ```Fabric``` or using the command line require ```Docker``` to be locally installed.
 
-If we use Mac or Windows we need the newer native client.
+If we use ```Mac``` or ```Windows``` we need the newer native client.
 
-To install docker we can run one of these commands:
+To install ```docker``` we can run one of these commands:
 
     fab docker.setup
     bin/docker_setup.sh
 
-You'll need to run ```docker login``` before trying any operation that involves pushing our images to Docker registry.
+You'll need to run ```docker login``` before trying any operation that involves pushing our images to ```Docker registry```.
```

Comment:
I understand the case with ` ```docker login``` ` but I'm not sure about **Docker registry**.
`` `Docker registry` `` - is a broad term it could be e.g. private, public, trusted.... I would not consider it **important** to bold it. It does not [give special importance or prominence to (something) in speaking or writing.](https://www.google.com/search?q=emphasize+definition&rlz=1C1JPGB_enLT722LT722&oq=emphasiz&aqs=chrome.3.69i57j0l5.8058j1j7&sourceid=chrome&ie=UTF-8)
I would use ** occasionally in cases like **NOTE:**, **Important:** where some special attention of a reader is required or some stress is needed on some keyword or process to understand its importance.

Other cases where I am not sure what to do below is my approach:
`` `.eyaml` ``  - file name extensions
`` `yaml` `` and `` `eyaml` `` -  backends  
`` `feature branch` `` 
`` `integration environment` ``
`` `Merge Request` ``
`` `pluginsync` ``  semi technical term, technology 
` ```my_password``` `  parameters

I do not see much value of marking "computer things" exclusively with ` ``` `.  Instead 
I would simply use single backticks everywhere and put extra word like directory, path, etc but ill try my best to follow your style guide.

Reply:
So, I agree on not emphasizing words like docker registry (I just took it as example to try to explain what should be printed with monospaced fonts, and what should be emphasized with bold).

I tend to use monospaced fonts , with 3 ticks , for:
- Any command , or command line
- Paths

I use four spaces indentation (so blocks of monospaced paragrphs) for:
- system's output text
- commands on multiple lines

I use asterisk (bold) to emphasize words.

A good example of this approach is on the main README: https://github.com/example42/psick